lua/telescope/mappings.lua

```diff
@@ -16,7 +16,7 @@ mappings.default_mappings = config.values.default_mappings or {
       ["<Down>"] = actions.move_selection_next,
       ["<Up>"] = actions.move_selection_previous,
 
-      ["<CR>"] = actions.goto_file_selection_edit,
+      ["<CR>"] = actions.goto_file_selection_edit + actions.center,
```

Comment:
Will remove this before merging like i said in my original post. Currently it is only there for testing purposes.
Would also wait until @smartding has confirmed that it really works :laughing:. 

If you wanna keep it we should do this for `normal` mapping as well. And maybe for `sp` and `vsp` as well.

Reply:
Confirmed working. `actions.center` is removed. Still would like to hear your thoughts about making `center` default or not.